### DISCUSSION TITLE
Grafana ssh access

### DISCUSSION THREAD

**Discussion by malosaa** [tteck/Proxmox#1369] [Help Needed] [answered]
Hi Tteck, 

I did install a long while ago grafana, and have everything running .
But as today i wanted to make a backup from my dashboards in case something happens.
SO i tried to access by ssh with wincsp lik i do with all my containers.
However this connects, but my password for root is wrong.. but i can login with root on my console | grafana container in proxmox.
The thing is i don't remember if i did run the script manual for install.. 

Whats the default pass in case i did run the script automatically instead of manual ?

Thanks and you really did a great job with all those scripts, i did fully move over to proxmox all because of your scripts!
👍

**Comment by tteck** [comment]
When you enter the LXC console, is there a prompt requesting login credentials?

**Reply by malosaa** [reply]
Its directly loggedin as root@grafana

![image](https://user-images.githubusercontent.com/13116501/234937078-3ae14cfa-c30e-455e-b835-170c8222b430.png)

**Reply by tteck** [reply, marked as answer]
You need to undo automatic login to get ssh working, https://github.com/tteck/Proxmox/discussions/324

**Reply by malosaa** [reply]
Many thanks for the fast reply, that solved my issue :)
